spec/lib/common/arp-spec.js

```diff
@@ -22,16 +22,13 @@ describe('ARPCache', function () {
     before(function () {
         arpCache = helper.injector.get('ARPCache');
         fs = helper.injector.get('fs');
-        sinon.stub(fs, 'readFileAsync');
     });
 
     beforeEach(function() {
-        fs.readFileAsync.reset();
+        this.sandbox.stub(fs, 'readFileAsync');
```

Comment:
I see that readFileAsync 's stub, restore, reset handling is not aligned in this PR